### PR TITLE
Fix registration bug #56

### PR DESCRIPTION
**Pull request by NovitskaMariia**
Added .env.example

Checked user creation with and without "profile_img_url"

Screenshot 2024-04-24 173023
Screenshot 2024-04-24 173134

### CODE REVIEW

File: startups/migrations/0001_initial.py
Comment (Thoughtseize1) [inline startups/migrations/0001_initial.py:1]: Delete migration file.

File: users/migrations/0001_initial.py
Comment (Thoughtseize1) [inline users/migrations/0001_initial.py:1]: Delete migration file

File: users/views.py [hunk 1]
```diff
@@ -66,14 +67,13 @@ def post(self, request):
 
                 token = RefreshToken.for_user(custom_user).access_token
                 current_site = get_current_site(request).domain
-                relative_link = reverse('send_email_confirmation')
-                abs_url = 'http://' + current_site + relative_link + '?token=' + str(token)
+                relative_link = reverse('verify-email', kwargs={'token': token, 'user_id': custom_user.id})
+                abs_url = 'http://'+ current_site + relative_link + '?token=' + str(token) + '?id=' + str(custom_user.id)
```
Comment (Thoughtseize1) [inline users/views.py:71]: Look, maybe we shoul use `&` for separate parameters.

**Review by Thoughtseize1 — CHANGES_REQUESTED** [review]

File: users/views.py (hunk 1 above)
Comment (NovitskaMariia) [reply users/views.py:71]: In my another PR I already changed abs_url
image

Comment (Thoughtseize1) [reply users/views.py:71]: Oh, see in future PR we change it to f-string method🙌🏻✅

**Review by Thoughtseize1 — APPROVED** [review]
Now it's clear.

File: .env.example [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# PostgreSQL settings
+POSTGRES_DB=fyour_DB_name
+POSTGRES_USER=your_postgress_user
+POSTGRES_PASSWORD=our_postgress_password
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
+
+# Django settings
+SECRET_KEY='django-insecure-t&h(6bczgs&7s8to0qp5i)csn_izs%=t!*q(km=fhk4l@$p2xl'
```
Comment (VovaD199) [inline .env.example:9]: I hope this is not a real SECRET_KEY

**Review by VovaD199 — APPROVED** [review]

File: .env.example (hunk 2 above)
Comment (NovitskaMariia) [reply .env.example]: Oooops. Already deleted!

**Pull request merged**